src/onnx/onnx_parser.cpp

```diff
@@ -238,6 +257,26 @@ void onnx_parser::parse_from(const void* data, std::size_t size)
     }
 }
 
+void onnx_parser::parse_opset_version(const onnx::ModelProto& model)
+{
+    const auto& opset_import = model.opset_import();
+    int64_t version          = -1;
+    for(const auto& opset : opset_import)
+    {
+        if(opset.has_version())
+        {
+            version = (version < opset.version()) ? opset.version() : version;
```

Comment:
can we rewrite this as `version = std::max(version, opset.version())`?

Reply:
changed